cloudinit/openvpn-config.tpl.yml

```diff
@@ -13,303 +13,119 @@ write_files:
-      # and DNS lookups to go through the VPN
-      # (The OpenVPN server machine may need to NAT
-      # or bridge the TUN/TAP interface to the internet
-      # in order for this to work properly).
-      ;push "redirect-gateway def1 bypass-dhcp"
-
-      # Certain Windows-specific network settings
-      # can be pushed to clients, such as DNS
-      # or WINS server addresses.  CAVEAT:
-      # http://openvpn.net/faq.html#dhcpcaveats
-      # The addresses below refer to the public
-      # DNS servers provided by opendns.com.
+      # Push DNS server and search path to clients
       push "dhcp-option DNS ${client_dns_server}"
       push "dhcp-option DOMAIN ${client_dns_search_domain}"
 
-      # Uncomment this directive to allow different
-      # clients to be able to "see" each other.
-      # By default, clients will only see the server.
-      # To force clients to only see the server, you
-      # will also need to appropriately firewall the
-      # server's TUN/TAP interface.
-      ;client-to-client
-
-      # Uncomment this directive if multiple clients
-      # might connect with the same certificate/key
-      # files or common names.  This is recommended
-      # only for testing purposes.  For production use,
-      # each client should have its own certificate/key
-      # pair.
-      #
-      # IF YOU HAVE NOT GENERATED INDIVIDUAL
-      # CERTIFICATE/KEY PAIRS FOR EACH CLIENT,
-      # EACH HAVING ITS OWN UNIQUE "COMMON NAME",
-      # UNCOMMENT THIS LINE OUT.
-      ;duplicate-cn
-
-      # The keepalive directive causes ping-like
-      # messages to be sent back and forth over
-      # the link so that each side knows when
-      # the other side has gone down.
-      # Ping every 10 seconds, assume that remote
-      # peer is down if no ping received during
-      # a 120 second time period.
+      # The keepalive directive causes ping-like messages to be sent back and
+      # forth over the link so that each side knows when the other side has gone
+      # down. Ping every 10 seconds, assume that remote peer is down if no ping
+      # received during a 120 second time period.
       keepalive 10 120
 
-      # For extra security beyond that provided
-      # by SSL/TLS, create an "HMAC firewall"
-      # to help block DoS attacks and UDP port flooding.
+      # The inactivity directive causes OpenVPN to exit after n seconds of
+      # inactivity on the TUN/TAP device. The time length of inactivity is
+      # measured since the last incoming or outgoing tunnel packet. The default
+      # value is 0 seconds, which disables this feature.  If the optional bytes
+      # parameter is included, exit if less than bytes of combined in/out
+      # traffic are produced on the tun/tap device in n seconds. In any case,
+      # OpenVPN’s internal ping packets (which are just keepalives) and TLS
+      # control packets are not considered “activity”, nor are they counted as
```

Comment:
Did you make sure that these fancy quotes don't break anything?  It might be safer to keep this file ASCII.